co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureMessageAmbassador.java

```diff
@@ -162,16 +164,16 @@ private synchronized void receiveInteraction(InfrastructureV2xMessageReception i
      * 
      *            Note: This function should be called after the
      *            onRsuRegistrationRequest
+     * @throws UnknownHostException
      */
-    private void onDsrcRegistrationRequest(String infrastructureId) {
+    private void onDsrcRegistrationRequest(String infrastructureId) throws UnknownHostException {
         // Create an InterfaceConfiguration object to represent the configuration of the
         // Ad-Hoc interface
         // Set the IP address and subnet mask to null for now
         // Set the transmit power to 50 dBm and the maximum range to 100 meters
-        // NOTE: TODO Setup the IP address and subnet
         InterfaceConfiguration interfaceConfig = new InterfaceConfiguration.Builder(AdHocChannel.SCH1)
-                .ip(null)
-                .subnet(null)
+                .ip( (Inet4Address) Inet4Address.getByName("192.168.0.1") ) //TODO: set IP address if needed
```

Comment:
When a new infrastructure registration message is received, the DsrcRegistrationRequest is triggered. Currently, this parameter is hard-coded because our need does not allow for its configuration. However, it is necessary for the DSRC initialize function.

Reply:
I think this is an easy ignore on the sonar scanner, this is a virtual IP (e.g. one within the simulated context) not an actual network IP. I can't yet tell exactly how this IP is used within the context (is it the self-assigned IP address of the radio?) but it shouldn't matter as DRSC isn't an IP protocol anyway, so this wouldn't be used even if we had better data for it.